apps/design/frontend/src/export_screen.test.tsx

```diff
@@ -36,59 +45,127 @@ function renderScreen() {
   );
 }
 
-test('export election package', async () => {
+test('export all ballots', async () => {
   renderScreen();
   await screen.findByRole('heading', { name: 'Export' });
 
-  apiMock.exportElectionPackage.expectCallWith({ electionId }).resolves({
+  apiMock.exportAllBallots.expectCallWith({ electionId }).resolves({
     zipContents: Buffer.from('fake-zip-contents'),
     electionHash: '1234567890abcdef',
   });
 
-  screen.getButton('Export Election Package').click();
```

Comment:
Wait, how did this `.click()` even work before? I didn't even know that was possible. And I supposedly wrote these tests?? 🥴

Reply:
Yeah definitely surprised me! I think it's just the native [HTMLElement click method](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click). Only reason I noticed this was because the tests started displaying "need `act` wrapping" warnings as I added state updates to the code